src/components/TodoLine/TodoLine.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { useAppDispatch } from '../../app/hooks';
+
+type Props = {
+  todo: Todo;
+  isSelected: boolean;
+};
+
+export const TodoLine: React.FC<Props> = ({ todo, isSelected }) => {
+  const selectedTodoDispatch = useAppDispatch();
+
+  const changeCurrentTodo = (newTodos: Todo) => {
+    selectedTodoDispatch(currentTodoSlice.actions.changecurrentTodo(newTodos));
```

Comment:
There is a typo in the action name `changecurrentTodo`. It should be `changeCurrentTodo` to match the typical camelCase convention and ensure consistency with the action defined in the `currentTodoSlice`.